spec/requests/api/v1/events/events_spec.rb

```diff
@@ -37,10 +35,13 @@
       lon: event.lon,
       name: event.name,
       started_at: event.started_at,
-      owner: {device_token: event.owner.device_token
+      owner: {
+        device_token: event.owner.device_token
       }
     }
-    post '/v1/events', { event: event_params }.to_json, 'Content-Type' => 'application/json'
+    post '/v1/events',
```

Comment:
You can also use the path helper here too. Less critical, but a good practice none the less